simulator/src/config_parser.py

```diff
@@ -84,7 +84,7 @@ def parse_config(config_file_path):
     else:
         sys.stdout.write(str(errors))
         sys.stdout.write("Please pass a Valid config file")
-        exit()
+        raise ValidationError(str(e))

```

Comment:
There is no variable called e

Here it should be:

ValidationError('Error validating config file - ' + str(errors))

sys.stdout can be removed